src/main/java/frc/robot/commands/ShallowClimbPrep.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.elevator.Elevator.ElevatorHeight;
+
+public class ShallowClimbPrep extends SequentialCommandGroup {
+    public ShallowClimbPrep(Elevator elevator){
+        addRequirements(elevator);
+        addCommands(
+            new InstantCommand(() -> elevator.setControlParams(false)),
```

Comment:
don't need to change controlParams in prep, only when we actually do the act of climbing in ShallowClimb